tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -231,7 +241,7 @@ std::unique_ptr<ROOT::Experimental::RNTupleModel>
 ROOT::Experimental::RNTupleModel::CreateBare(std::unique_ptr<RFieldZero> fieldZero)
 {
    auto model = std::unique_ptr<RNTupleModel>(new RNTupleModel(std::move(fieldZero)));
-   model->fProjectedFields = std::make_unique<RProjectedFields>(model.get());
+   model->fProjectedFields = std::make_unique<Internal::RProjectedFields>(*model);
```

Comment:
Is there an advantage of `*model` vs `model.get()`?

Reply:
`*model` dereferences the `std::unique_ptr`, `.get()` gets you a raw pointer - this is for the commit that takes references where a `nullptr` doesn't make sense.